tests/conftest.py

```diff
@@ -169,40 +177,72 @@ def average_per_body_complex_multi_body():
 
 
 @dataclasses.dataclass
```

Comment:
We don't need the results per body for elemental results => For elemental results only the combined reference files are used.